cmd/run/run.go

```diff
@@ -200,6 +200,8 @@ func NewRunCommand() *cobra.Command {
 	// Unfortunately UintSlice/IntSlice does not work well when used as environment variable, we need to stick with string slice and convert back to integer
 	flags.StringSlice("request-duration-datastore-query-count-buckets", defaultConfig.RequestDurationDatastoreQueryCountBuckets, "datastore query count buckets used in labelling request duration by query count histogram")
 
+	flags.StringSlice("request-duration-datastore-dispatch-count-buckets", defaultConfig.RequestDurationDatastoreDispatchCountBuckets, "datastore query count buckets used in labelling request duration by dispatch count histogram")
+
```

Comment:
Can we update the description for both of these? We don't track this metric for every API, only for the APIs tagged with "relationship queries" (https://openfga.dev/api/service#/Relationship%20Queries). _Note to self... we need to track this for Expand..._ 

Also, people reading these descriptions will probably not know what "dispatch count" means. I suggest: `buckets used in labelling request duration by dispatch count (i.e. Check subproblem count) histogram"` or something similar.

Reply:
I have updated the description but calling it "Check subproblem count" feels misleading as 
1. We will do this for ListObjects as well
2. We should have a common consistent term used everywhere and just dispatch count works for that.

Modified this a little bit